docs/modules/ROOT/pages/exit.adoc

```diff
@@ -0,0 +1,21 @@
+= exit
+
+Exit command exits with the given code.
```

Comment:
Exits with the specified code

Reply:
https://github.com/hazelcast/clc-docs/pull/5/commits/156434bc9a0369bc6690f0c7071ef6942ee01478